Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -598,6 +599,7 @@ async def long_running_loop():
         try:
             check_for_mirrors()
             check_for_unanswered_questions()
+            await asyncio.sleep(5)
```

Comment:
It's an async sleep to free up the cpu to perform other tasks. Since it loops continually, if there is no sleep, then it will occupy the process and block the other coroutines from executing.